pandas/tseries/resample.py

```diff
@@ -1204,6 +1206,13 @@ def _get_time_delta_bins(self, ax):
         end_stamps = labels + 1
         bins = ax.searchsorted(end_stamps, side='left')
 
+        if ax.hasnans:
```

Comment:
though we actually need to ignore the nans, doesn't this actually create a NaT group? this is inconcistent with grouping.